src/_guides/json.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "JSON Support"
+description: "Dart solutions for reading and writing JSON."
+---
+
+Most mobile and web apps use JSON for tasks such as
+exchanging data with a web server.
+This page discusses Dart support for JSON _serialization_ and _deserialization_:
+converting Dart objects _to_ and _from_ JSON.
+
+
+## Generally useful libraries
+
+The following libraries and packages are useful across Dart platforms:
+
+* [dart:convert](/guides/libraries/library-tour#dartconvert---decoding-and-encoding-json-utf-8-and-more)<br>
+  Converters for both JSON and UTF-8
+  (the character encoding that JSON requires).
+
+* [package:json_serializable](https://pub.dartlang.org/packages/json_serializable)<br>
+  Easy-to-use support for using the Dart build system to generate code that
+  serializes and deserializes Dart objects.
```

Comment:
How about something like this:

> An easy-to-use code generation package provided as a builder. Add metadata annotations to your classes, and the builder generates the JSON serialization and deserialization code for you.

We might want to link "builder", but I'm unsure where atm. (Maybe the new, forthcoming "build_runner" www page?

Reply:
> We might want to link "builder", but I'm unsure where atm. (Maybe the new, forthcoming "build_runner" www page?)

Sure, if it has a builder section. (Which I guess it should.) I'll update the bug about that page.